src/pages/Schedule.js

```diff
@@ -17,17 +20,8 @@ export default class Schedule extends Component {
     })
```

Comment:
I just realized ... why is there arrow function in setState? it is unnecessary, you can just pass an object to setState fn